components/tidb_query/src/rpn_expr/impl_math.rs

```diff
@@ -1026,6 +1043,17 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_rand() {
+        let got = RpnFnScalarEvaluator::new()
```

Comment:
`Rand()` two times and test if they're different in case we accidentally give a constant seed to it.

Reply:
added.